muxink/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "muxink"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bincode = { version = "1.3.3", optional = true }
+bytes = "1.1.0"
+futures = "0.3.21"
+serde = { version = "1.0.138", optional = true }
+thiserror = "1.0.31"
+tokio = { version = "1", features = [ "full" ] }      # TODO: Reduce features.
```

Comment:
While we're here - do you plan to reduce these features as part of the PR?

Reply:
Good point, fixed in 8c6374d77.